src/fides/api/task/task_resources.py

```diff
@@ -75,8 +67,6 @@ def build_connector(  # pylint: disable=R0911,R0912
             return BigQueryConnector(connection_config)
         if connection_config.connection_type == ConnectionType.saas:
             return SaaSConnector(connection_config)
-        if connection_config.connection_type == ConnectionType.manual:
-            return ManualConnector(connection_config)
```

Comment:
the v1 manual connector which assumed sequential execution is no longer supported, as noted elsewhere